cmd/infracost/main.go

```diff
@@ -150,3 +152,13 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func get_cwd() string {
+	cwd, err := os.Getwd()
+	if err != nil {
+		log.Warn(err)
+		cwd = "UNKNOWN"
```

Comment:
```suggestion
		cwd = ""
```
Any benefit of having "UNKNOWN" over just a blank string? We check for a blank string when validating the args here: https://github.com/infracost/infracost/blob/master/internal/providers/terraform/provider.go#L23

Reply:
not particularly; i don't tend to like blank strings b/c i like to interpolate values into log messages, and blank strings don't interpolate as helpfully as placeholders like "UNKNOWN", but this is a weak preference.